postinstall.js

```diff
@@ -0,0 +1,4 @@
+/* eslint-disable flowtype/require-valid-file-annotation */
+
+require('./symlink.js');
```

Comment:
think we should only require this if the script is executed inside of the `assets/admin` folder